getting-started/support_matrix.md

```diff
@@ -25,6 +25,8 @@ Amazon Linux 2 currently is shipped with SELinux as the LSM (Linux Security Modu
 
 The latest versions of Amazon Linux 2 ship with a new LSM type called BPF-LSM and Kubearmor [intends](https://github.com/kubearmor/KubeArmor/issues/484) to support it soon).
 
+Update: On Amazon Linux 2 with Kernel Version 5.10, Kubearmor is found to be running on systemd mode. The enforcement is sometimes partial with following supported LSM's: "Capability,Yama,Lockdown,SELinux,BPF". Refer [Issue](https://github.com/kubearmor/KubeArmor/issues/919).
+
```

Comment:
Hey @HariVamsiK, what does it mean by "enforcement is sometimes partial"?

Reply:
The issue was solved when I installed a previous version of kubearmor first, then removed it and installed the new version and doing a daemon reload. The command "kubearmor probe" gave the output as enforcement as partial when I installed the new version directly.

Reply:
Okay, the enforcement as Partial is not correct. In Amazon Linux-2, AppArmor is not available and in `karmor probe` we are only watching for AppArmor. We should ideally be also watching for BPF LSM too and we already have an issue for that here: https://github.com/kubearmor/kubearmor-client/issues/211 
Except that, in the screenshot you shared, KubeArmor is not running in the first place. The probe is just giving a note that enforcement will work if KubeArmor runs successfully.
Now with issue #919 user mentioned that they were not able to run KubeArmor v0.6 on AL-2. Can you confirm if you were able to successfully deploy KubeArmor (v0.6) or the latest version in systemd mode?

Reply:
I can confirm that I was able to successfully deploy KubeArmor(v0.6) in systemd mode. The following screenshot depicts the same. 
![2](https://user-images.githubusercontent.com/112711197/207092223-ae13944e-15ba-4245-8e12-54a2799c2fd2.jpg)

Reply:
Yeah it's working with the latest version of kubearmor V0.7.2
![4](https://user-images.githubusercontent.com/112711197/207096862-c59ce6c3-bf6c-4e2e-ad92-988fa206a88c.jpg)